public/css/login.css

```diff
@@ -0,0 +1,85 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  align-self: center;
+}
+
+.header {
+  width: 100%;
+  height: 40px;
+  background-color: #752223;
+  display: flex;
+  justify-content: flex-end;
+  margin-bottom: 20px;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+  width: 400px;
+}
+
+.h1 {
+  color: #752223;
+  font-family: Roboto;
+}
+
+.input {
+  margin-bottom: 10px;
+  height: 50px;
+  border-radius: 10px;
+  border-style: solid;
+  border-width: thin;
+  padding-left: 10px;
+  border-color: #a3a3a3;
+  font-size: 17px;
+}
+
+.button {
+  height: 45px;
+  border-radius: 10px;
+  border: none;
+  height: 55px;
+  background-color: #752223;
+  color: white;
+  font-weight: bold;
+  font-size: 18px;
+}
+
+.button:hover {
+  background-color: #631a1b;
+  cursor: pointer;
+}
+
+.a {
+  text-decoration: none;
+  color: #cf0606;
+}
+
+.h4 {
+  margin-top: 25px;
+  text-align: center;
+}
+
+.footer {
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  align-self: flex-end;
```

Comment:
you have to display flex before using flex properties :smile_cat:

Reply:
The line below has `display: flex;` so this is fine :+1:

Reply:
Cool, don't effect if this line have before/after >display: flex; ?